src/services.yml

```diff
@@ -1,9 +0,0 @@
-services:
-  psr_container:
-    class: RoaveBehatPsrContainer
-    factory: ['Roave\BehatPsrContainer\ContainerFactory', 'createContainerFromIncludedFile']
-    tags:
-      - {'name': 'helper_container.container'}
-    shared: false
-    scope: prototype
```

Comment:
It's the same as `shared: false` which maps to the `setShared()` call.

Apparently all versions of Symfony since 2.8 support this 'shared' thing rather than the old scope, so it doesn't need to be mapped.